lib/ui/SpinniesManager.ts

```diff
@@ -191,54 +229,57 @@ class SpinniesManager {
     }, interval);
   }
 
-  setStreamOutput(frame = '') {
+  private setStreamOutput(frame = ''): void {
     let output = '';
-    const linesLength = [];
+    const linesLength: number[] = [];
     const hasActiveSpinners = this.hasActiveSpinners();
-    Object.values(this.spinners).map(
-      ({
-        text,
+    Object.values(this.spinners).forEach(spinner => {
+      let { text } = spinner;
+      const {
         status,
         color,
         spinnerColor,
         succeedColor,
         failColor,
-        succeedPrefix,
-        failPrefix,
-        indent,
-      }) => {
-        let line;
-        let prefixLength = indent || 0;
-        if (status === 'spinning') {
-          prefixLength += frame.length + 1;
-          text = breakText(text, prefixLength);
-          line = `${chalk[spinnerColor](frame)} ${
-            color ? chalk[color](text) : text
-          }`;
+        indent = 0,
+        succeedPrefix = prefixOptions(this.options).succeedPrefix!,
```

Comment:
Would also just double check that these new defaults don't change any behavior here